yarn.lock

```diff
@@ -1893,15 +1893,22 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@devcycle/bucketing-assembly-script@^1.0.19, @devcycle/bucketing-assembly-script@workspace:lib/shared/bucketing-assembly-script":
+"@devcycle/bucketing-assembly-script@npm:^1.0.19":
+  version: 1.0.19
+  resolution: "@devcycle/bucketing-assembly-script@npm:1.0.19"
+  checksum: 372ce516181891da556a899ee3428df3a426fa841c871a193dbf34eba6e38bbb134d067b5b8ae939075f006e5157ca2bf714c378a907de76bd007d039738b35a
+  languageName: node
+  linkType: hard
+
+"@devcycle/bucketing-assembly-script@workspace:lib/shared/bucketing-assembly-script":
   version: 0.0.0-use.local
   resolution: "@devcycle/bucketing-assembly-script@workspace:lib/shared/bucketing-assembly-script"
   dependencies:
-    "@devcycle/bucketing-test-data": ^1.0.11
+    "@devcycle/bucketing-test-data": ^1.0.12-alpha.0
```

Comment:
@kaushalkapasi I assume the test data needs the same thing?

Reply:
hm.. yeah, might as well 👍